archspec/cpu/microarchitecture.py

```diff
@@ -218,15 +229,9 @@ def satisfies_constraint(entry, version):
             min_version, max_version = entry["versions"].split(":")
 
             # Check version suffixes
-            min_version, min_suffix = version_components(min_version)
-            max_version, max_suffix = version_components(max_version)
-            version, suffix = version_components(version)
-
-            # If the suffixes are not all equal there's no match
-            if (suffix != min_suffix and min_version) or (
-                suffix != max_suffix and max_version  # pylint: disable=bad-continuation
-            ):
-                return False
```

Comment:
@alalazo Can you clarify why this was stripped out?

Reply:
See https://github.com/archspec/archspec/pull/19#discussion_r440771243